controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -391,7 +434,7 @@ func (r *StorageConsumerReconciler) reconcileCephRadosNamespace(
 	var combinedErr error
 	for idx := range blockPools.Items {
 		bp := &blockPools.Items[idx]
-		if bp.Name == "builtin-mgr" {
+		if bp.Name == "builtin-mgr" || bp.Name != util.GenerateNameForCephNFSBlockPool(storageCluster) {
```

Comment:
What is this change about?

Reply:
The radosnamespace that we create should not be created for `builtin-mgr`, and the blockpool that we have for nfs.